### DISCUSSION TITLE
[macOS] Tried splitting and compressing, and and after a while I got a sound and the progress bar became empty

### DISCUSSION THREAD

**Discussion by andrew1-1** [aonez/Keka#1429] [Q&A]
<img width="516" alt="Screenshot 2024-04-10 at 13 10 36" src="https://github.com/aonez/Keka/assets/35116193/a0aa8103-4021-419b-b8c1-19d7868b8c61">

Is it normal? Does it mean the compression was successful?

**Comment by aonez** [comment]
Hi @andrew1-1,

The process ended successfully but this is not the normal behavior, the window should have been closed automatically.

I’ll try to replicate. What version are you using? Was that a very quick operation or rather a lengthy one?

**Comment by andrew1-1** [comment]
The operation took around 2 minutes.
The version I was using was the 1.3.8 (5320).

By the way, I deleted the app and then reinstalled it (same version) and retried the same split (same size splitting) and compression with the same file, and this time the progress bar simply caused itself when the process completed (which is its normal expected behaviour, isn't it).

**Reply by andrew1-1** [reply]
Update: It happened again with a different file. Same operation, i.e. splitting and compressing, and same "issue", i.e. progress window becomes empty and stays open when process finishes instead of closing itself.

**Comment by aonez** [comment]
@andrew1-1 please enable the debug logging, both `Enable` and `Save` options.  As soon as your reproduce it, whenever it is, open the logs folder and attach the log files here (you can compress them in a ZIP file). Do you have any third party application that controls window behaviour, like Magnet, Rectangle or similar?

<img width="571" alt="Screenshot 2024-04-12 at 15 03 12" src="https://github.com/aonez/Keka/assets/480163/cb2faac5-3c79-4328-a5f1-b7f53a2c6f49">
